### PR TITLE
fix parse address

### PR DESCRIPTION
**Pull request by WangXiangUSTC**

### CODE REVIEW

File: pkg/utils/urls_test.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		"pump-1:8250,pump-2:8250",
+		"http://127.0.0.1:2379",
+		"https://127.0.0.1:2379",
+		"http://127.0.0.1:2379,http://127.0.0.2:2379",
+		"https://127.0.0.1:2379,https://127.0.0.2:2379",
+		"unix:///home/tidb/tidb.sock",
+	}
+
+	for _, url := range urls {
+		_, err := ParseHostPortAddr(url)
```
Comment (july2993) [inline pkg/utils/urls_test.go]: check return value

File: pkg/utils/urls.go [hunk 2]
```diff
@@ -50,5 +52,6 @@ func ParseHostPortAddr(s string) ([]string, error) {
 		addrs = append(addrs, u.String())
 	}
 
+	fmt.Println(addrs)
```
Comment (july2993) [inline pkg/utils/urls.go]: remove

**Comment by IANTHEREAL** [comment]
LGTM

**Comment by WangXiangUSTC** [comment]
@july2993 PTAL

**Review by july2993 — APPROVED** [review]
can't tell what this func ParseHostPortAddr really want to do...

**Comment by WangXiangUSTC** [comment]
check the urls is valid or not, and split it to several url @july2993

**Pull request merged**